src/Admin/AdminHelper.php

```diff
@@ -77,6 +78,8 @@ public function getChildFormView(FormView $formView, string $elementId): ?FormVi
      *
      * @throws \RuntimeException
      * @throws \Exception
+     *
+     * @phpstan-return array{\Sonata\AdminBundle\Admin\FieldDescriptionInterface|null, \Symfony\Component\Form\FormInterface}
```

Comment:
it is strange to not use uses, this line is on 3.x?

Reply:
They are, but they were added this way https://github.com/sonata-project/SonataAdminBundle/commit/bb7b9ae624336902faed4a12592209d11a599a12#diff-b195c94401e2b41ba855367436ab8899b22acca2641057d5361fdb5d35a75da2R111.

Reply:
Phpstorm consider `use` as unused since he doesn't understand the annotation

Reply:
Well if it is on 3.x fine by me, but IMO the not supported by phpstorm is not strong argument to not use the short class name